applications/build-modular-monolith-with-platformatic/services/media-service/plugin.js

```diff
@@ -0,0 +1,69 @@
+/// <reference path="./clients/people/people.d.ts" />
+
+'use strict'
+
+/** @param {import('fastify').FastifyInstance} app */
+module.exports = async function peopleDataPlugin (app) {
+  function buildOnComposerResponseCallback (peopleProps) {
+    return async function addPeopleToResponse (request, reply, body) {
+      let entities = await body.json()
+
+      const multipleEntities = Array.isArray(entities)
+      if (!multipleEntities) {
+        entities = [entities]
+      }
+
+      const peopleIds = []
+      for (const entity of entities) {
+        for (const { idProp } of peopleProps) {
+          peopleIds.push(entity[idProp])
+        }
+      }
+
+      const people = await app.people.getPeople({ "where.id.in": peopleIds.join(',') })
+
+      const getPersonNameById = (id) => {
+        const person = people.find(person => person.id === id)
+        return (person) ? person.name : null
+      }
+
+      for (let entity of entities) {
+        for (const { idProp, nameProp } of peopleProps) {
+          entity[nameProp] = getPersonNameById(entity[idProp])
+        }
+      }
+
+      reply.send(multipleEntities ? entities : entities[0])
```

Comment:
```suggestion
      return reply.send(multipleEntities ? entities : entities[0])
```

@ivan-tymoshenko is it expected to call `reply.send()` here?